editor/filesystem_dock.cpp

```diff
@@ -2120,6 +2136,84 @@ void FileSystemDock::set_file_list_display_mode(FileListDisplayMode p_mode) {
 	_toggle_file_display();
 }
 
+Control *FileSystemDock::create_tooltip_for_path(const String &p_path) const {
+	if (DirAccess::exists(p_path)) {
+		// No tooltip for directory.
+		return nullptr;
+	}
+
+	HBoxContainer *hb = memnew(HBoxContainer);
+
+	VBoxContainer *vb = memnew(VBoxContainer);
+	vb->set_alignment(BoxContainer::ALIGNMENT_CENTER);
+	vb->add_theme_constant_override("separation", -4 * EDSCALE);
+	{
+		Label *label = memnew(Label(p_path.get_file()));
+		vb->add_child(label);
+	}
+
+	{
+		Ref<FileAccess> f = FileAccess::open(p_path, FileAccess::READ);
+		Label *label = memnew(Label(TTR("Size:") + " " + String::humanize_size(f->get_length())));
+		vb->add_child(label);
+	}
+
+	if (ResourceLoader::exists(p_path)) {
+		String type = ResourceLoader::get_resource_type(p_path);
+		{
+			Label *label = memnew(Label(TTR("Type:") + " " + type));
+			vb->add_child(label);
+		}
+
+		Ref<Texture2D> preview;
+
+		if (ClassDB::is_parent_class(type, "Image")) {
+			Ref<Image> image = ResourceLoader::load(p_path);
+			if (image.is_valid()) {
+				preview = ImageTexture::create_from_image(image);
+
+				Label *label = memnew(Label(TTR("Dimensions:") + vformat(" %d x %d", preview->get_width(), preview->get_height())));
```

Comment:
```suggestion
				Label *label = memnew(Label(TTR("Dimensions:") + vformat(String::utf8(" %s×%s"), preview->get_width(), preview->get_height())));
```

See https://github.com/godotengine/godot/pull/60008#discussion_r847653853

Reply:
I would suggest moving the translation into the vformat call. So that the colon & space after it can be properly localized. e.g. CJK usually prefer removing the space after a full-width colon.

```suggestion
				Label *label = memnew(Label(vformat(TTR("Dimensions: %d×%d"), preview->get_width(), preview->get_height())));
```

Same to other hardcoded whitespace here.